src/win32/pthread/pthread_mutex_init.c

```diff
@@ -39,7 +39,9 @@
 #include <Objbase.h>
 #include <assert.h>
 
+#if REALM_UWP
 int _getpid(void);
+#endif
```

Comment:
Is this file being compiled for anything but UWP now? If so, does this not change the old behaviour?